cli/README.md

```diff
@@ -245,7 +260,7 @@ Once your app has been configured to query for updates against the CodePush serv
 
 2. [React Native](#releasing-updates-react-native) - Performs the same functionality as the general release command, but also handles the task of generating the updated app contents for you (JS bundle and assets), instead of requiring you to run both `react-native bundle` and then `code-push release`.
 
-3. [Cordova](#releasing-updates-cordova) - Performs the same functionality as the general release command, but also handles the task of preparing the app update for you, instead of requiring you to run both `cordova prepare` and then `code-push release`.
+3. [Cordova](#releasing-updates-cordova) - Performs the same functionality as the general release command, but also handles the task of preparing the app update for you, instead of requiring you to run both `cordova prepare` (or `phonegap build` for PhoneGap instances)  and then `code-push release`.
```

Comment:
This should be `phonegap prepare`, and we can probably remove the `for PhoneGap instances` section.